### ISSUE TITLE
Hi, how to use multi gpus?

### ISSUE THREAD

**Issue by louielu1027** [ZFTurbo/Keras-RetinaNet-for-Open-Images-Challenge-2018#13]
I use 8 gpus for infenrence, but it is slowly than 1 gpu......
I changed  'gpu_use = 0' to   'gpu_use = 0,1,2,3,4,5,6,7' in retinanet_inference_example.py.
Is it right?

**Comment by ZFTurbo** [comment]
No, this variable only controls which GPU use for calculation. Code is for single GPU only. If you need inference on several GPU, just split data in N parts and run inference in parallel on each of them.

**Comment by louielu1027** [comment]
Ok, Thanks!
